libcoap-sys/build/build_system/vendored.rs

```diff
@@ -314,7 +350,7 @@ impl VendoredBuildSystem {
                 .context("DEP_OPENSSL_INCLUDE has no parent directory")?
                 .join("lib");
 
-            // Just add the OpenSSL directory to the PKG_CONFIG_PATH, that way libcoap will find it.
+            // Set pkg-config path for version and library/include path determination.
```

Comment:
Is this change related to WolfSSL?

Reply:
Not strictly, but I used this wording in the corresponding wolfSSL function and found it better, so I also applied it to the OpenSSL code.